src/components/directory-content.js

```diff
@@ -625,6 +704,20 @@ const DirectoryContent = () => {
         );
     };
 
+    const allowsCopyFilterToScript = () => {
```

Comment:
Seems like is the same allowing function than `allowsCopyContingencyToScript`, you could add a OR condition in `allowsCopyToScript`. No need to manage diferent allowing here for now at least.